example/models.py

```diff
@@ -3,6 +3,9 @@
 
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
+from django.contrib.contenttypes.models import ContentType
```

Comment:
After reviewing this package's code, imports are usually alphabetically sorted. Could you move these `contrib` imports about the `db` import?